jupyter_kernel_test/__init__.py

```diff
@@ -57,9 +57,11 @@ def test_kernel_info(self):
 
 
     def execute_helper(self, code, timeout=TIMEOUT,
-                       silent=False, store_history=True):
+                       silent=False, store_history=True,
+                       stop_on_error=True):
         msg_id = self.kc.execute(code=code, silent=silent,
-                                 store_history=store_history)
+                                 store_history=store_history,
+                                 stop_on_error=True)
```

Comment:
This should be `stop_on_error=stop_on_error` - otherwise it's ignoring the parameter passed in ;-)